docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: ./Dockerfile.development
+    command: bash -c "/app/bin/wait-for-it.sh db:5432 -- ./bin/download_geolite2.sh;./bin/runsslserver.sh"
+    links:
+      - db
+    networks:
+      - public_nw
+    volumes:
+      - .:/app
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@db/normandy
```

Comment:
FYI: changing this to `postgres://postgres:postgres@db/postgres` fixed it for me

Reply:
I think 189c849 might've fixed it too?

Reply:
yup